src/sys/windows/named_pipe.rs

```diff
@@ -0,0 +1,610 @@
+//! Windows named pipes bindings for mio.
+//!
+//! This crate implements bindings for named pipes for the mio crate. This
```

Comment:
Outdated docs and you might want to move them to the `NamePipe` type as that is the only thing that is exported.